### DISCUSSION TITLE
Shorter way to change station

### DISCUSSION THREAD

**Discussion by WladiSGE** [Suplanus/Cuterdio#105] [Ideas]
Hallo, erstmal danke für die tolle app. Sie ist kurz davor meine aktuelle Lieblingsapp (instaradio) als Standardapp im Auto abzulösen :)) Ich habe nur einen Wunsch, könnte man irgendwie, per Wisch oder über einen Pfeil direkt aus der Standardansicht des Senders (das Einhornbild/Senderlogo während des Abspielens), den Sender wechseln ohne extra dafür in die Senderübersicht zu gehen?

Also 1 Klick statt 2 Klicks. Ich weiß im Auto gibt es einen Knopf dafür aber wenn das man das Programm nicht gerade im Auto nutzt, ist es umständlich, das Menu zu öffnen und dann den Sender zu wechseln. Gerne auch als eine Option, d.h. wer das nicht will kann dies in Einstellungen abschalten.

Nutze die App mit IOS 13.6, Iphone XS. Die Betaversion über Testflight.

Danke, 
Viele Grüße, Wladi

**Comment by Suplanus** [comment]
Danke erstmal :)
In der aktuellen Beta kannst du per Swipe direkt in die Senderübersicht gehen und somit ist es 1 Swipe + 1 Tap.

So wie ich es verstehe würdest du gerne einen Swipe auf das Cover machen um den Sender zu wechseln?

**Comment by WladiSGE** [comment]
Danke für die schnelle Antwort. Genau, wenn man einfach hin und herschalten möchte ist dieser zusätzlicher Swipe schon ein Aufwand. Gerne Swipe auf Cover oder ein zusätzlicher Senderwechsel knopf :)
